docs/Basic-Guides.md

```diff
@@ -0,0 +1,141 @@
+# Basic Guides
+
+This guide shows you the basic steps you can try to play around with ML-agents. 
```

Comment:
This page contains several introductory guides to quickly get you started using ML-Agents.